### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Documentation changes

* **What is the current behavior?** (You can also link to an open issue here)
[README](https://github.com/pythonindia/inpycon-blog/blob/master/README.md)

* **What is the new behavior (if this is a feature change)?**
Updated the readme to include contribution instructions for writers and also updated the installation instructions.
Should we also use Tags and Summary for the blog metadata, or should I remove them?

### CODE REVIEW

**Comment by Sc0rpi0n101** [comment]
CC: @vipulgupta2048

**Comment by vipulgupta2048** [comment]
Reviewed it back on Friday but forgot to hit finalise review button.

File: README.md [hunk 1]
```diff
@@ -2,32 +2,44 @@
...
 
-To create a new blog, create a markdown file (.md) in the `content/blog` directory. For Pelican to parse the content properly, make sure the Metadata syntax at the start of your new blog is follows the pattern given below:
+3. To create a new blog, create a markdown file (.md) in the `content/blog` directory. For Pelican to parse the content properly, make sure the Metadata syntax at the start of your new blog is follows the pattern given below:
 
 ```
 Title: Announcing X as Titanium Sponsor
 Date: 2020-05-10 10:00
 Author: John Doe
 Category: 2020
+Tags: pelican, publishing
```
[the remark this answers is not in the document]
Comment (Sc0rpi0n101) [reply README.md:40]: not sure, that's why i asked

[the remark this answers is not in the document]
Comment (vipulgupta2048) [reply README.md:42]: At the moment we don't support tags. We are going to, but till then I feel we shouldn't include this.

**Comment by Sc0rpi0n101** [comment]
Sure, I'll do that

**Pull request merged**